.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports =  {
+    semi:  false,
+    trailingComma:  "none",
+    singleQuote:  true,
+    printWidth:  120,
```

Comment:
I think it was 80?

Reply:
Can't really find where it was 80 in the file history. But the default value is 80 so I will change it anyways. 👌

Reply:
Yes, it was not set, so it was the default.